addons/io_scene_gltf2/blender/exp/gltf2_blender_extract.py

```diff
@@ -408,10 +409,14 @@ def __get_positions(blender_mesh, key_blocks, armature, blender_object, export_s
 
 def __get_normals(blender_mesh, key_blocks, armature, blender_object, export_settings):
     """Get normal for each loop."""
-    blender_mesh.calc_normals_split()
-
     normals = np.empty(len(blender_mesh.loops) * 3, dtype=np.float32)
-    blender_mesh.loops.foreach_get('normal', normals)
+    if key_blocks:
+        normals = key_blocks[0].relative_key.normals_split_get()
+        normals = np.array(normals)
```

Comment:
This makes a float64 array by default. Pass `dtype=np.float32` to get a float32 array, like the other case.